R/utils-parse-expr.R

```diff
@@ -1,8 +1,28 @@
 #' Parse dynamic dots into a list of expressions (PlRExpr, not polars-expr)
 #' @noRd
 parse_into_list_of_expressions <- function(..., `__structify` = FALSE) {
-  list2(...) |>
-    lapply(\(x) as_polars_expr(x, structify = `__structify`)$`_rexpr`)
+  dots <- list2(...)
+  not_spliced <- vapply(
+    dots,
+    \(x) {
+      if (!is_bare_list(x)) {
+        return(FALSE)
+      }
+      elem_are_expr <- vapply(x, \(elem) is_polars_expr(elem), FUN.VALUE = logical(1))
+      any(elem_are_expr)
+    },
```

Comment:
This is more complicated than what I wanted, but we need to accept lists that contain only characters or numeric values for instance (see the new tests for some examples).

Reply:
In my opinion this is too complicated and too expensive.
Can't we simply use `try_fetch` to catch errors and display the message?